src/applications/edu-benefits/1990/helpers.jsx

```diff
@@ -2,6 +2,51 @@ import React from 'react';
 
 import { transformForSubmit } from 'us-forms-system/lib/js/helpers';
 
+export function prefillTransformer(pages, formData, metadata) {
+
+  let newContactInfo = {};
+
+  if (formData && formData.email) {
+    newContactInfo = {
+      email: formData.email,
+      'view:confirmEmail': formData.email
+    };
+  }
+
+  if (formData && formData.homePhone) {
+    newContactInfo = {
+      ...newContactInfo,
+      homePhone: formData.homePhone
+    };
+  }
+
+  if (formData && formData.mobilePhone) {
```

Comment:
You also should remove these contact fields from the top level of the form data object you return. I think they'll get overwritten on submit, but we shouldn't leave them hanging around.